src/messaging/network.rs

```diff
@@ -54,17 +54,37 @@ pub enum NodeTransferCmd {
 
 ///
 #[allow(clippy::large_enum_variant)]
-#[derive(Debug, Hash, Eq, PartialEq, Clone, Serialize, Deserialize)]
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
 pub enum NodeDataCmd {
-    ///
+    /// Duplicate a given chunk at another Adult
     DuplicateChunk {
-        ///
+        /// New Holder's name
         new_holder: XorName,
-        ///
+        /// Address of the blob to be duplicated
         address: BlobAddress,
-        ///
+        /// Current Holders
+        fetch_from_holders: BTreeSet<XorName>,
+    },
+    /// Get Chunk from current holders for duplication
```

Comment:
Is this _only_ for duplication purposes? ie. no blob requests would use this? Just wondering on naming here. It's maybe too broad atm?